example/starwars/starwars.go

```diff
@@ -4,12 +4,14 @@
 package starwars
 
 import (
+	"context"
```

Comment:
Changes here are just for demonstration purposes if someone wants to play with the feature a bit. To try it out, run the starwars example server and perform this `curl` call:

```
curl 'http://localhost:8080/query' -d '{"query":"query {search(text:\"C-3PO\") {... on Droid {name} ... on Starship {name}}}","variables":null}'
```

Reply:
I do not intend to ship that if the feature is merged, it will be removed.